src/interface/apiRequest.ts

```diff
@@ -12,9 +12,27 @@ export class RegisterUserRequest {
 }
 
 export class CreateFolderRequest {
-  @ApiProperty({ type: String })
+  @ApiProperty()
   location: string;
 
-  @ApiProperty({ type: String })
+  @ApiProperty()
   name: string;
 }
+
+export class DeleteFolderRequest {
+  @ApiProperty()
+  path: string;
+}
+
+export class UploadFileRequest {
+  @ApiProperty()
+  location: string;
+
+  @ApiProperty()
+  fileInfos: Array<{ name: string; size: number }>;
```

Comment:
Please, check swagger. This api property is displayed as an array of strings there. It can be fixed by setting params for ApiProperty decorator. Check the docs, please.